### DISCUSSION TITLE
Username error

### DISCUSSION THREAD

**Discussion by KristopherJnr** [community/community#29173] [New to GitHub]
Hi guys, my github username keeps showing not found when i try to link my account on other platforms

**Comment by ghost** [comment]
I need help with this too....if you found a way out, please share with me.

**Reply by KristopherJnr** [reply]
What i did was remove the '@' when typing the username. Just type it as a normal text without "@" at the beginning.

Lemme know if it works for you.

**Comment by melvincwng** [comment]
@KristopherJnr and @Wanjikutheuri , for account specific issues, please raise a ticket to [GitHub Support](https://support.github.com/contact/account) over here.

They are the best team to handle such account related issues. Kindly describe your problem to them over a ticket, and wait for a few working days for the support engineers to investigate your issue.

Hope this clarifies.

**Reply by KristopherJnr** [reply]
Thanks, i have resolved my issue.

**Comment by ghost** [comment]
Thanks.  The issue was resolved.

**Reply by KristopherJnr** [reply]
How did you do it?

**Comment by ghost** [comment]
I wrote to them and luckily they responded fast.
